### PR TITLE
Add header with Hello Microverse!

### PR DESCRIPTION
**Pull request by norbiboy1**
Checkpoints:
✅ linters configuration;
✅ html file with Hello Microverse title;
✅ css file with colored title.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (shella12) [inline .gitignore:1]: - [ ] Great job adding a `.gitignore` file, and including `node_modules` into it, Please also add `test.md` into the file as well .  For this project your `.gitignore` should have two files, for example:

<kbd>
Screenshot 2023-04-26 at 7 33 34 AM
</kbd>

File: README.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[key_feature_1]**
+- **[key_feature_2]**
+- **[key_feature_3]**
+
```
Comment (shella12) [inline README.md]: - [ ] Great job on using Microverse template however I would suggest you to customise the template content to reflect your project and delete any irrelevant content from the template that doesn't apply to your project 

_Note: Removing the irrelevant content doesn't mean deleting any of the main sections, it's just reference to the predefined example content inside each section _

File: README.md [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
```
Comment (shella12) [inline README.md]: - [ ] Please remove the default comments like 

> Add a link to your deployed project.

> Write a message to encourage readers to support your project  

> Describe 1 - 3 features you will add to the project.  ,

`etc` from your README. They were used in the template to explain to the user the purpose of each section. You can go ahead and remove it from your README

File: README.md [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (shella12) [inline README.md]: - [ ] Please remove Note from the LICENCE section of the README. Note is there in the template to guide you on how you to add a LICENCE to your README. Once you have successfully added the LICENCE you can go ahead and remove it

File: README.md [hunk 5]
```diff
@@ -0,0 +1,216 @@
...
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (shella12) [inline README.md]: - [ ] I would suggest adding at least 1 - 3 future features that you will add to the project. This is the requirement of the project to add and customise all the main sections from the Microverse Template

File: README.md [hunk 6]
```diff
@@ -0,0 +1,216 @@
...
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- **[Question_1]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
```
Comment (shella12) [inline README.md]: - [ ] Kindly complete and customise the `support` and `acknowledgement` section of the README. Also FAQ section of the README is optional if you don't want to customise it please remove it from your README.

**Review by shella12 — CHANGES_REQUESTED** [review]
Hi @norbiboy1 , 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- All linters passed ✅ 
- Correct use of git flow ✅ 
- Good `feature branch `name ✅ 
- Good commit messages ✅ 
- Descriptive pull request ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ABDULALI3468 — APPROVED** [review]
Hi @norbiboy1 , 👍 

# APPROVED 💯 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights 
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by norbiboy1** [comment]
Thank you so much! Now it looks the readme so professional! 😃

**Pull request merged**